test/acceptance/cli-test.js

```diff
@@ -442,6 +442,37 @@ describe('ember-template-lint executable', function () {
     });
   });
 
+  describe('multiplatform config resolve', function () {
+    describe('given absolute config path and path to single file without errors', function () {
```

Comment:
Question, is this different from the existing tests for `--config-path`? Does it actually test a Windows-style path like `C:\Users\...`. And does it fail before the fix?

Reply:
it should test windows-style path in our windows CI env
no failures before

Reply:
also, it should fail here: https://github.com/ember-template-lint/ember-template-lint/blob/master/test/helpers/fake-project.js#L68
but not failing, we could experiment with some prefixes ('/' before 'C'), or node versions to catch this error

Reply:
in original bugreport #2336 it's failing for custom plugins (as I understand), likely we could also try to test custom plugin here..